client/modules/Mobile/MobileSketchView.jsx

```diff
@@ -82,6 +84,7 @@ const MobileSketchView = (props) => {
           clearConsole={clearConsole}
         />
       </Content>
+      <Footer before={<Console />} />
```

Comment:
Rather than this being `before`, could this not just be
```jsx
<Footer>
  <Console />
</Footer>
```
??

Reply:
I think you fixed this in `MobileIDEView` but not here??

Reply:
How did I miss that twice?? Anyway, it should be OK now